### DISCUSSION TITLE
Import geometry

### DISCUSSION THREAD

**Discussion by thsmit** [PMEAL/OpenPNM#2425] [Q&A] [answered]
Hi, 

Question: how can I import a geometry from a stl or vtk file for analysis?

**Comment by jgostick** [comment, marked as answer]
I will need a bit more context.  I *think* you are referring to a 3D geometry like something you'd use for 3D printing.  This is not really compatible with Openpnm.  The only way to get this into openpnm would be to convert your STL into a voxel image, the use porespy to extract a network from the image, THEN you can open it in openpnm.

**Reply by thsmit** [reply]
Oke, that answers my question thanks!
